docs/configuration.adoc

```diff
@@ -138,6 +138,36 @@ IMPORTANT: This strategy is currently only applicable for _white box_ testing ap
 
 WARNING: At this moment, this strategy does not work with Java 9.
 
+===== Explicitly Set
+
+By default affected strategy uses _imports_ of tests to build the graph of dependencies.
+This approach is fine for unit tests (white box tests) but might not work in all cases of high level tests (black box test).
+
+There are some test technologies that allows you to deploy an application locally and then run tests against it.
+For example Wildfly Swarm has `@DefaultDeployment` annotation or for example Spring (Boot) deploys all application automatically.
+
+This means that production classes are not directly imported into the test, so there is no way to get a relationship between test and production classes.
+For this reason `affected` provides an annotation to set package(s) of production classes that are deployed by the test.
+
+The first thing you need to do is register next artifact into your build script: `org.arquillian.smart.testing:api:<version>`.
+
+Then you can annotate your test with `org.arquillian.smart.testing.strategies.affected.Tests` annotation.
```

Comment:
Could be awesome to extract `org.arquillian.smart.testing.strategies.affected.Tests` automatically from the code - sounds like small doable extension?

Reply:
I am not sure if small but yes, let me open a new issue, since we are using the same approach in several places and several files I think it is better to implement as a separate PR.

Reply:
Yes, by all means, do so. I didn't mean to have it as part of this PR.

Reply:
https://github.com/arquillian/smart-testing/issues/256